build.sbt

```diff
@@ -285,6 +283,20 @@ lazy val semanticdbScalac = project
       }).transform(node).head
     }
   )
+  .dependsOn(semanticdbScalac)
+
+
+lazy val semanticdbScalac = project
+  .in(file("scalameta/semanticdb-scalac"))
+  .settings(
+    moduleName := "semanticdb-scalac-core",
+    description := "Scala 2.x compiler plugin that generates semanticdb on compile",
```

Comment:
`Library to generate semanticdb from Scala 2.x internal data structures`

Reply:
Done.